_opensearch/rest-api/cat/cat-segments.md

```diff
@@ -46,6 +46,7 @@ In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensear
 Parameter | Type | Description
 :--- | :--- | :---
 bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/)..
+cluster_manager_timeout | Time | The amount of time to wait for a connection to the master node. Default is 30 seconds.
```

Comment:
"cluster_manager" node?

Reply:
In the description, should it be with or without underscore? 

The amount of time to wait for a connection to the cluster_manager node           vs
The amount of time to wait for a connection to the cluster manager node

Reply:
In #513 it says "master" is replaced with "cluster manager." So, I will change to "cluster manager" without the underscore unless there are any objections. Thank you!